thunder/dynamo/splitter.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import annotations
 from typing import TYPE_CHECKING
+import copy
```

Comment:
where is this used? I don't see any use in diff

Reply:
forgot to remove it
```suggestion
```